.htmlhintrc

```diff
@@ -13,9 +13,9 @@
   "doctype-html5": true,
   "id-class-value": false,
   "style-disabled": true,
-  "inline-style-disabled": true,
+  "inline-style-disabled": false,
```

Comment:
There are [some disadvantages](https://www.linkedin.com/advice/3/what-risks-using-inline-styles-html-skills-web-development-kip2c#:~:text=One%20of%20the%20biggest%20problems,external%20or%20internal%20style%20sheet.) with using inline styling. Is there any good reason we now want to use it?

Reply:
@victorjbassey there are a lot of inline style already in the project, so to prevent Sonar failure i intend to disable it and enable it back once I'm done with the translations.